src/renderer/components/ft-list-lazy-wrapper/ft-list-lazy-wrapper.js

```diff
@@ -86,18 +79,23 @@ export default defineComponent({
           return false
         }
       } else if (data.type === 'channel') {
-        if (this.channelsHidden.includes(data.channelID) || this.channelsHidden.includes(data.name)) {
+        if (this.channelsHidden.includes(data.id) || this.channelsHidden.includes(data.name)) {
           // hide channels by author
           return false
         }
       } else if (data.type === 'playlist') {
-        if (this.channelsHidden.includes(data.authorId) || this.channelsHidden.includes(data.author)) {
+        if (this.channelsHidden.includes(data.channelId) || this.channelsHidden.includes(data.channelName)) {
```

Comment:
I'm guessing this pull request was only tested with the local API as Invidious uses authorId and author, which you have removed instead of adding additional checks.